packages/table/src/table.tsx

```diff
@@ -1156,6 +1156,12 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         e.stopPropagation();
 
         const { focusedCell } = this.state;
+        if (focusedCell == null) {
```

Comment:
Try keyboard-arrowing around in the `selectedRegionTransform` example on `/table/preview`. That's where the errors were showing up for me.